e2e/clone.go

```diff
@@ -0,0 +1,84 @@
+package e2e
```

Comment:
probably should add a copyright/license header?

Reply:
we dont have copyright/license for the e2e tests

Reply:
but we should?

Reply:
will add to all the e2e files in follow-up.